feat: Include theme colors in theme object (#29)
posit-dev/py-shinyswatch: CHANGELOG.md

## [UNRELEASED]

### New features

> * Themes in `shinyswatch.theme` are now callable classes. You no longer need to call the theme as a function, although existing code will still work. The theme object now includes a `.colors` attribute with the theme's color palette, including colors like `body_color`, `body_bg`, `primary`, `secondary`, etc.  You can use these colors to theme plots, outputs and other UI elements to match the shinyswatch theme. (#29)

Reviewer: ```suggestion
* Themes in `shinyswatch.theme` are now callable classes. You no longer need to call the theme as a function, although existing code will still work. E.g. `shinyswatch.theme.yeti` (#29)

* `shinyswatch.theme`'s theme object now includes a `.colors` attribute with the theme's color palette, including colors like `body_color`, `body_bg`, `primary`, `secondary`, etc.  You can use these colors to theme plots, outputs and other UI elements to match the shinyswatch theme. (#29)
```

Author: ```suggestion
* Themes in `shinyswatch.theme` are now callable classes. You no longer need to call the theme as a function, e.g. `shinyswatch.theme.yeti`. Existing code calling the theme, e.g. `shinyswatch.theme.yet()` will continue to work. (#29)

* `shinyswatch.theme`'s theme object now includes a `.colors` attribute with the theme's color palette, including colors like `body_color`, `body_bg`, `primary`, `secondary`, etc.  You can use these colors to theme plots, outputs and other UI elements to match the shinyswatch theme. (#29)
```

> 

Reviewer: ```suggestion

### Updates
```
